### PR TITLE
updated iiot entry redirection

### PR DESCRIPTION
**Pull request by gatemezing**

### CODE REVIEW

File: iiot/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+https://w3id.org/IIoT is a permanent URI 
+
+for the IIoT ontology for Industrial Internet of Things: 
+
+https://ontoit.github.io/IIoT/IIoT.owl
+
+Maintainer: Liviana Tudor
```
Comment (TallTed) [inline iiot/readme.md:7]: @dgarijo — You did not request inclusion of Liviana Tudor's email address nor github ID?

**Comment by davidlehn** [comment]
I'm not sure what's going on here, but I'm going to revert this now.  These `iiot` contents are a duplicate of `IIoT`.  And we have a policy to avoid case similar ids due to confusion in some file systems.

File: iiot/readme.md (hunk 1 above)
Comment (gatemezing) [reply iiot/readme.md:7]: tudor.liviana@gmail.com - @OntoIT

**Comment by dgarijo** [comment]
@gatemezing look at the folder names you are uploading. You have to keep consistent with the case sensitive systems.

**Comment by gatemezing** [comment]
Probably I’m using a different OS.
My understanding was to have only one “IIoT” with the new redirections.
Is it something possible ?



"*Love all, trust a few, do wrong to none*" (W. Shakespeare)
Web: http://atemezing.org



Le ven. 18 oct. 2024 à 08:56, Daniel Garijo ***@***.***> a
écrit :

**Comment by dgarijo** [comment]
This PR adds a new namespace `iiot`. It does not rename anything.

**Comment by dgarijo** [comment]
In order to simplify things, would it be ok if we only kept `iiot` (non-capital letters)?

**Comment by gatemezing** [comment]
> In order to simplify things, would it be ok if we only kept `iiot` (non-capital letters)?
This is more for you @OntoIT - could you answer?

**Pull request merged**